docs/toc.yml

```diff
@@ -0,0 +1,3 @@
+- name: API Documentation
+  href: ../site/yml/api/
+  homepage: ../site/yml/api/Gossip.Configuration.yml
```

Comment:
This feels a bit suboptimal, since if we change this namespace name or delete the namespace altogether, this will break, right?